### ISSUE TITLE
ZTE Majesty Pro Plus (Z899VL) missing from list

### ISSUE THREAD

**Issue by speck1234** [deadman96385/zte_programmers#1]
XDA Forum thread tutorial on bootloader unlocking has the Z899VL in the list of known working models, but the QFIL firehose for that model isn't shown here. I have looked all over the internet for it

**Comment by BananaDoppio** [comment]
Yes i have been searching too and this is the only place that i thought it would be but it is missing
